pkg/apis/admissionregistration/v1/defaults.go

```diff
@@ -27,7 +27,8 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-func SetDefaults_ValidatingWebhook(obj *admissionregistrationv1.ValidatingWebhook) {
```

Comment:
The name of defaulting methods is fixed and cannot be changed without also changing the generated defaulters. That is not currently planned, due to the wide ripples in all codebases that depend on generated defaults.

Reply:
The changes on names of method are all removed now.